### PR TITLE
SEO housekeeping

### PR DESCRIPTION
**Pull request by radkostanev**
Related to https://github.com/telerik/blazor/issues/8069

### CODE REVIEW

File: components/dropdownlist/overview.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 title: Overview
 page_title: DropDownList Overview
-description: Discover the Blazor DropdownList for Blazor and explore the examples.
+description: Explore the Blazor DropdownList component and see code examples. Read more in the Telerik UI documentation for Blazor.
```
Comment (ntacheva) [inline components/dropdownlist/overview.md:4]:
```suggestion
description: Explore the Blazor DropDownList component and see code examples. Read more in the Telerik UI documentation for Blazor.
```

File: components/dropdownlist/overview.md [hunk 2]
```diff
@@ -44,29 +44,33 @@ Selected value: @selectedValue
...
 ````
 
+>caption The rendered DropDownList component from the code snippet above:
+
+![Blazor DropDown List component](images/dropdownlist-basic-screenshot.jpg)
+
 ## Data Binding
 
-The Blazor DropDownList @[template](/_contentTemplates/dropdowns/features.md#data-binding) [Read more about the Blazor DropDownList data binding...]({% slug components/dropdownlist/databind %}).
+The Blazor dropdown list @[template](/_contentTemplates/dropdowns/features.md#data-binding) [Read more about the Blazor dropdown list component data binding...]({% slug components/dropdownlist/databind %}).
```
Comment (ntacheva) [inline components/dropdownlist/overview.md:53]: This contradicts the model we are currently following - using the actual component name. If I properly understand, we should use the separate keywords "dropdown" and "list" but I suggest not plugging them in every component mention.

**Review by ntacheva — APPROVED** [review]

File: components/dropdownlist/overview.md (hunk 1 above)
Comment (radkostanev) [reply components/dropdownlist/overview.md]: Addressed through ea8366c9a0b2aeeeada1d9fc662ed794a55740b9

File: components/dropdownlist/overview.md (hunk 2 above)
Comment (radkostanev) [reply components/dropdownlist/overview.md]: Concerns addressed through ea8366c9a0b2aeeeada1d9fc662ed794a55740b9

File: components/dropdownlist/overview.md [hunk 3]
```diff
@@ -44,29 +44,33 @@ Selected value: @selectedValue
...
 
 ## Filtering
 
-The Blazor DropDownList @[template](/_contentTemplates/dropdowns/features.md#filtering) [Read more about the Blazor DropDownList filter...]({% slug components/dropdownlist/filter %}).
+The Blazor dropdown list @[template](/_contentTemplates/dropdowns/features.md#filtering) [Read more about the Blazor dropdown list component filter...]({% slug components/dropdownlist/filter %}).
 
 ## Grouping
 
-The Blazor DropDownList @[template](/_contentTemplates/dropdowns/features.md#grouping) [Read more about the Blazor DropDownList grouping...]({% slug components/dropdownlist/grouping %}).
+The Blazor dropdown list @[template](/_contentTemplates/dropdowns/features.md#grouping) [Read more about the Blazor dropdown component grouping...]({% slug components/dropdownlist/grouping %}).
```
[the remark this answers is not in the document]
Comment (radkostanev) [reply components/dropdownlist/overview.md]: Concerns addressed through ea8366c9a0b2aeeeada1d9fc662ed794a55740b9

**Review by yordan-mitev — APPROVED** [review]

**Pull request merged**